cmd/loom/loom.go

```diff
@@ -337,6 +339,17 @@ func newRunCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "run [root contract]",
 		Short: "Run the blockchain node",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			output, err := exec.Command("ulimit", "-n").Output()
```

Comment:
We should not rely on external linux command like ulimit. Instead, we should take a look at `syscall` library. Take a look at go-ethereum https://github.com/ethereum/go-ethereum/tree/fdb34b7a7c87142cde48a2ec092f77a56d66f773/common/fdlimit

Also, we should try to raise the file descriptor limit automatically instead of asking the users to manually do so.